src/wwwroot/template-editor/adapterService.js

```diff
@@ -340,40 +353,10 @@
     function getSettings(idCampaign, idTemplate) {
       traceAdapterCall(getSettings, arguments);
       var msEditorSettings = {
-        language: 'es',
-        sharedSocialNetworks: [{
-          name: 'twitter',
-          url: 'http://twitter.com/share?related=fromdoppler',
-          idSocialNetwork: '0'
-        }, {
-          name: 'linkedin',
-          url: 'http://linkedin.com/',
-          idSocialNetwork: '1'
-        }, {
-          name: 'facebook',
-          url: 'http://facebook.com',
-          idSocialNetwork: '2'
-        }, {
-          name: 'googlemas',
-          url: 'http://google.com',
-          idSocialNetwork: '3'
-        }, {
-          name: 'pinterest',
-          url: 'http://pinterest.com',
-          idSocialNetwork: '4'
-        }, {
-          name: 'whatsapp',
-          url: 'http://whatsapp.com',
-          idSocialNetwork: '5'
-        }],
-        stores: [{
```

Comment:
Will it work or do you need to return an empty array for _stores_?

Reply:
We will return a empty array of stores and we will update MSEditor logic to handle sharedSocialNetworks when is empty.